lib/ansible/plugins/connection/netconf.py

```diff
@@ -5,68 +5,118 @@
 from __future__ import (absolute_import, division, print_function)
 __metaclass__ = type
 
+
 DOCUMENTATION = """
-    author: Ansible Networking Team
-    connection: netconf
-    short_description: Use netconf to run command on network appliances
+---
+author: Ansible Networking Team
+connection: netconf
+short_description: Provides a persistent connection using the netconf protocol
+description:
+  - This plugin actually forces use of 'local' execution but uses paramiko to
+    establish a remote ssh netconf session on the network device.  This connection
+    plugin will load an additional plugin to manage the session.  It will
+    load a netconf plugin to handle serializing requests to the remote device.
+version_added: "2.3"
+options:
+  host:
     description:
-        - Use netconf to run command on network appliances
-    version_added: "2.3"
-    options:
-      network_os:
-        description:
-            - Appliance specific OS
-        default: 'default'
-        vars:
-            - name: ansible_netconf_network_os
-      password:
-        description:
-            - Secret used to authenticate
-        vars:
-            - name: ansible_pass
-            - name: ansible_netconf_pass
-      private_key_file:
-        description:
-            - Key or certificate file used for authentication
-        vars:
-            - name: ansible_private_key_file
-            - name: ansible_netconf_private_key_file
-      ssh_config:
-        type: boolean
-        default: False
-        description:
-            - Flag to decide if we use SSH configuration options with netconf
-        vars:
-            - name: ansible_netconf_ssh_config
-        env:
-            - name: ANSIBLE_NETCONF_SSH_CONFIG
-      user:
-        description:
-          - User to authenticate as
-        vars:
-          - name: ansible_user
-          - name: ansible_netconf_user
-      port:
-        type: int
-        description:
-          - port to connect to on the remote
-        default: 830
-        vars:
-          - name: ansible_port
-          - name: ansible_netconf_port
-      timeout:
+      - Specifies the remote device FQDN or IP address to establish the SSH
+        connection to.
+    default: inventory_hostname
+    vars:
+      - name: ansible_host
+      - name: ansible_ssh_host
```

Comment:
http://docs.ansible.com/ansible/latest/intro_inventory.html states

> Ansible 2.0 has deprecated the “ssh” from ansible_ssh_user, ansible_ssh_host, and ansible_ssh_port to become ansible_user, ansible_host, and ansible_port. If you are using a version of Ansible prior to 2.0, you should continue using the older style variables (ansible_ssh_*). These shorter variables are ignored, without warning, in older versions of Ansible.


So should `ansible_ssh_host` be deleted, ditto for `ansible_ssh_port`, etc?